_pages/business-units/18f/leadership-resources/leadership-selection.md

```diff
@@ -64,7 +64,7 @@ When you’ve finished all of the above steps, here’s what you should have at
 
 Self-nomination is great for people who don’t mind promoting themselves, but not everyone feels comfortable with self-promotion. Relying solely on self-nomination is inequitable because it disadvantages people who come from cultural backgrounds that promote modesty or only receiving recognition through the work done rather than self-promotion. To help ensure leadership roles are accessible to everyone, we must also nominate or encourage people to apply.
 
-If you want to nominate or “nudge” someone to apply, first think carefully about why you want to nominate them. Look at the position description and role responsibilities, and think about specific, concrete ways you believe the person will be successful. If someone is not well-suited for a particular role, don’t nominate them just because they’re your friend or you admire them, and certainly not just because they are from a community that is currently underrepresented in leadership.
+If you want to nominate or “nudge” someone to apply, first think carefully about why you want to nominate them. Look at the position description and role responsibilities, and think about specific, concrete ways you believe the person will be successful. If someone is not well suited for a particular role, don’t nominate them just because they’re your friend or you admire them, and certainly not just because they are from a community that is currently underrepresented in leadership.
```

Comment:
Since this isn't an instance of a compound adjective preceding the noun it modifies, I removed the hyphen.